deploy/kubernetes/dolphinscheduler/templates/svc-dolphinscheduler-api.yaml

```diff
@@ -40,6 +40,13 @@ spec:
       {{- end }}
       protocol: TCP
       name: api-port
+    - port: 25333
+      targetPort: python-api-port
+      { { - if and (eq .Values.api.service.type "NodePort") .Values.api.service.nodePort } }
```

Comment:
`{ {` is wrong in Helm Chart syntax, please remove the space between the 2 `{`

Reply:
> `{ {` is wrong in Helm Chart syntax, please remove the space between the 2 `{`

Done